backend/src/extension.ts

```diff
@@ -35,7 +35,8 @@ export function activate(context: vscode.ExtensionContext) {
 	context.subscriptions.push(vscode.commands.registerCommand("yeomanUI.toggleOutput", YeomanUIPanel.toggleOutput));
 
 	vscode.window.registerWebviewPanelSerializer(YeomanUIPanel.viewType, {
-		async deserializeWebviewPanel(webviewPanel: vscode.WebviewPanel, state: any) {
+		async deserializeWebviewPanel(webviewPanel: vscode.WebviewPanel, state?: any) {
+			_.set(state, "filter", _.get(state, "genFilter"));
```

Comment:
no need for -> _.set(state, "filter", _.get(state, "genFilter"));
already have genFilter on state obj.

Reply:
UI command passes only filter not genFilter
That is why when we try to rectreate UI we need to pass filter